types/proto3/block.proto

```diff
@@ -1,7 +1,7 @@
 syntax = "proto3";
 
 package proto3;
```

Comment:
why not `tendermint.proto3`?

Reply:
I wanted not to break everything that's why I didn't prefix the other packages but yes it's definitely better to namespace everything, I can add this.